tests/test_main.py

```diff
@@ -114,7 +114,7 @@ def test_post_identity_with_traits(
     response = client.post(
         "/api/v1/identities/",
         headers={"X-Environment-Key": environment_key},
-        data=json.dumps(data),
+        content=orjson.dumps(data),
```

Comment:
Sorry, this line and [this other one](https://github.com/Flagsmith/edge-proxy/pull/77/files#diff-41d180f6f7233d175c2dfe36c45c4ea80eed4754fa7ed4eed46ccde1b2a778c7L142) should've been submitted with #76. I'm cherry-picking most of these changes, and I missed these two.